package.json

```diff
@@ -15,6 +15,8 @@
     "react-router-dom": "^6.4.3",
     "react-scripts": "5.0.1",
     "uuid": "^9.0.0",
+    "redux-thunk": "^2.4.2",
```

Comment:
[OPTIONAL] You are free to remove this [line](https://github.com/ivanmvh/bookstore-ivanmvh/blob/BR5-BOOKSTORE-CONNECT-REDUX-TO-API/README.md#:~:text=NOTE%3A%20we%20recommend%20using%20the%20MIT%20license%20%2D%20you%20can%20set%20it%20up%20quickly%20by%20using%20templates%20available%20on%20GitHub.%20You%20can%20also%20use%20any%20other%20license%20if%20you%20wish.) as well.

Reply:
I don't undestand this remove line comment @Rustamxon7 . Thanks.